pom.xml

```diff
@@ -28,6 +28,8 @@
     </modules>
 
     <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Isn't this change also in [#76](https://github.com/vaadin/appsec-kit/pull/76/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R25-R26)? Let's remove it from here to target just the NPE with this PR.

Reply:
Removed.